meson.build

```diff
@@ -38,7 +38,7 @@ if compiler_id == 'clang'
   global_link_args += [ '-lstdc++', '-lm' ]
 endif
 
-add_global_arguments(global_args, language : 'cpp')
+add_global_arguments(global_args,['-Wno-ignored-qualifiers'], language : 'cpp')
```

Comment:
During the compilation of `mi_bf`, `sdsl` library is used and compiler throws and error for `sdsl` library that would require changing the source code. As I wont change the `sdsl` library source code, I silenced this error.

Reply:
A better way to solve is to change sdsl dependency line to:
```
sdsl_dep = sdsl_subproject.dependency('sdsl', include_type: 'system'))
```
The `include_type: 'system'` will make the compiler skip warnings for the dependency.

Reply:
Thanks, I changed as suggested.